packages/combobox/src/index.tsx

```diff
@@ -508,6 +530,8 @@ export const ComboboxInput = forwardRefWithAs<ComboboxInputProps, "input">(
         aria-controls={listboxId}
         aria-expanded={isVisible}
         aria-haspopup="listbox"
+        aria-label={ariaLabel}
+        aria-labelledby={ariaLabelledby}
         role="combobox"
         {...props}
```

Comment:
Worth pointing out that the `aria-label` and `aria-labelledby` props can still be applied directly to `<ComboboxInput>` through `props`, so this doesn't break existing uses of Combobox.